server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailSubmissionSetMethodFutureReleaseContract.scala

```diff
@@ -297,8 +299,10 @@ trait EmailSubmissionSetMethodFutureReleaseContract {
       .body(request)
       .post
 
-    // Wait one second
-    updatableTickingClock.setInstant(DATE.plusSeconds(1))
+    // Wait ten seconds
+    updatableTickingClock.setInstant(DATE.plusSeconds(10))
+
+    Thread.sleep(5000)
```

Comment:
Can we find other solutions than such a looooong wait time? Why do we need to wait for?

Reply:
Should I decrease the waiting time? I try to sleep 1s and tests are still correct